### PR TITLE
add browser cache policy on docker images

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Related to #2637

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

- [x] Tested by quality team

### CODE REVIEW

**Comment by PhilippeAccorsi** [comment]
Need information in changelog to explain what is necessary to use this new config file on existing installation of tracim with docker.

Related to #2571

**Comment by PhilippeAccorsi** [comment]
Tested with new build of docker images.

File: backend/doc/apache.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Install `Apache` server and uWSGI:
+
+    sudo apt install apache2 libapache2-mod-wsgi-py3 uwsgi uwsgi-plugin-python3
+
+### Configuration ###
+
+Create a file named `/etc/apache2/sites-available/tracim.conf` containing: 
+
+    <VirtualHost *:80>
+        ServerName tracim
```
Comment (inkhey) [inline backend/doc/apache.md]: replace tracim by [server_name] (and explain that user need to change it, tracim is domain_name).

File: backend/doc/apache.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Enable this configuration file:
+
+    sudo ln -s /etc/apache2/sites-available/tracim.conf /etc/apache2/sites-enabled/tracim.conf
+
+
+Create the file named `/etc/uwsgi/apps-available/tracim.ini` containing:
+
+    [uwsgi]
+    plugins = python3
```
Comment (inkhey) [inline backend/doc/apache.md]: add virtualenv parameter, by default using setup_default_backend.sh , we need to use virtualenv (/backend/env ?), look in tracim v1 documentation.

File: backend/doc/apache.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply backend/doc/apache.md]: done

File: backend/doc/apache.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply backend/doc/apache.md]: Done

**Comment by PhilippeAccorsi** [comment]
Tested on branch.

**Pull request merged**